### PR TITLE
Document where to find the legal settings links in the ownCloud UI

### LINKED ISSUE #572: Add pic for imprint in ownCloud
## What Needs to be Documented?
We need a pic of how the imprint is displayed in ownCloud

## Where Does This Need To Be Documented?
https://doc.owncloud.com/server/administration_manual/configuration/server/legal_settings_configuration.html

https://github.com/owncloud/docs/edit/master/modules/administration_manual/pages/configuration/server/legal_settings_configuration.adoc

## Why Should This Change Be Made? (Optional)

Caused customer confusion.

## What Type Of Content Change Is This? (Optional)
- [x] New Content Addition
- [ ] Old Content Deprecation
- [ ] Existing Content Simplification
- [ ] Bug Fix to Existing Content

## Which Manual Does This Relate To? (Optional)
- [x] Admin Manual
- [ ] Developer Manual
- [ ] User Manual
- [ ] Android
- [ ] iOS
- [ ] Branded Clients
- [ ] Desktop Client
- [ ] Other

---------
### Example

impressum2

### PR DESCRIPTION
**Pull request by settermjd**
This PR updates the Legal Settings section of the Administration Manual to show where the imprint and privacy URLs are displayed, when configured.

This fixes #572.

### CODE REVIEW

**Comment by voroyam** [comment]
can't see the picture that was uploaded.

**Comment by voroyam** [comment]
seen the picture.

This https://github.com/owncloud/docs/issues/572 can be closed if this is merged

**Comment by settermjd** [comment]
@voroyam, just updated this quite a bit to cover the discussion that you, @pmaier1, and I were having.

**Comment by voroyam** [comment]
Okay, still missing the email templates, or am I not seeing something?

**Comment by settermjd** [comment]
I didn't add a reference to them, after the final word from @pmaier1, or did I misunderstand his intent?

File: modules/administration_manual/pages/configuration/server/legal_settings_configuration.adoc [hunk 1]
```diff
@@ -1,6 +1,13 @@
 = Legal Settings Configuration
+:experimental:
+:release-notes-imprint-and-privacy-url: https://doc.owncloud.com/server/administration_manual/release_notes.html#new-options-to-display-imprint-and-privacy-policy
+
+Because of one or more legal frameworks around the world, some ownCloud instances may need to display links to both an Imprint as well as a Privacy Policy on all pages (both in the Web UI and within email templates).
```
Comment (pmaier1) [inline modules/administration_manual/pages/configuration/server/legal_settings_configuration.adoc]: Important detail: Imprint and Privacy Policy are shown on all **public** pages and in the footer of e-mails. Authenticated pages like files app or settings do not show them.

**Pull request merged**